src/router/storeSync/LayerParamConfig.class.js

```diff
@@ -127,19 +127,19 @@ export function createLayerObject(parsedLayer, currentLayer, store, featuresRequ
                 .toString()
                 .split(':')
                 .forEach((featureId) => {
-                    featuresRequests.push(
-                        getFeature(
-                            store.getters.getLayerConfigById(parsedLayer.id),
-                            featureId,
-                            store.state.position.projection,
-                            {
+                    const layer = store.getters.getLayerConfigById(parsedLayer.id)
+                    if (layer.isHighlightable) {
+                        // only highlightable feature will output something, for the others a click coordinate is required
+                        // (and we don't have it if we are here, as we are dealing with pre-selected feature in the URL at app startup)
```

Comment:
So, I know `layer` is already used outside of the scope but
I would move this out of the forEach, and have at line 124 somthing like :
`124 : const layerConfig = store.getters.getLayerConfigById(parsedLayer.id)`
`125: if (features !== undefined && layerConfig.isHighlightable) { ... `
It doesn't change that much, I must concede that, but I do find it a bit cleaner

Reply:
`layer` was already computed above, so I removed the `.getters.` call in the for loop and used the already present variable instead

Reply:
that's even better then